File: base/src/main/vsdk/toolkit/render/shaders/GouraudTextureShader.java
package vsdk.toolkit.render.shaders;

// GLSL analogue: gouraudTexturePixelShader.glsl
public final class GouraudTextureShader extends LightingShader {
    public GouraudTextureShader(boolean textureEnabled)
    {
        super(true, textureEnabled, false);
    }
}

File: typescript/base/src/main/vsdk/toolkit/render/shaders/GouraudTextureShader.ts
import { LightingShader } from "./LightingShader.js";

// GLSL analogue: gouraudTexturePixelShader.glsl
export class GouraudTextureShader extends LightingShader {
    public constructor(textureEnabled: boolean) {
        super(true, textureEnabled, false);
    }
}
